examples/3d/postprocessing.rs

```diff
@@ -0,0 +1,214 @@
+/// arrow keys are held.
+const CHROMATIC_ABERRATION_INTENSITY_ADJUSTMENT_SPEED: f32 = 0.002;
+
+/// The maximum supported chromatic aberration intensity level.
+const MAX_CHROMATIC_ABERRATION_INTENSITY: f32 = 0.4;
+
+/// The settings that the user can control.
+#[derive(Resource)]
+struct AppSettings {
+    /// The intensity of the chromatic aberration effect.
+    chromatic_aberration_intensity: f32,
+}
+
+/// The entry point.
+fn main() {
+    App::new()
+        .init_resource::<AppSettings>()
+        .add_plugins(DefaultPlugins.set(WindowPlugin {
+            primary_window: Some(Window {
+                title: "Bevy Chromatic Aberration Example".into(),
+                ..default()
+            }),
+            ..default()
+        }))
+        .add_systems(Startup, setup)
+        .add_systems(Update, handle_keyboard_input)
+        .add_systems(
+            Update,
+            (update_chromatic_aberration_settings, update_help_text).after(handle_keyboard_input),
+        )
+        .run();
+}
+
+/// Creates the example scene and spawns the UI.
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>, app_settings: Res<AppSettings>) {
+    // Create the scene.
+    spawn_scene(&mut commands, &asset_server);
+
+    // Spawn the camera.
+    spawn_camera(&mut commands, &asset_server);
+
+    // Spawn the help text.
+    spawn_text(&mut commands, &app_settings);
+}
+
+/// Spawns the scene.
+///
+/// This is just the tonemapping test scene, chosen for the fact that it uses a
+/// variety of colors.
+fn spawn_scene(commands: &mut Commands, asset_server: &AssetServer) {
+    // Spawn the main scene.
+    commands.spawn(SceneBundle {
+        scene: asset_server.load(
+            GltfAssetLabel::Scene(0).from_asset("models/TonemappingTest/TonemappingTest.gltf"),
+        ),
+        ..default()
+    });
+
+    // Spawn the flight helmet.
+    commands.spawn(SceneBundle {
+        scene: asset_server
+            .load(GltfAssetLabel::Scene(0).from_asset("models/FlightHelmet/FlightHelmet.gltf")),
+        transform: Transform::from_xyz(0.5, 0.0, -0.5)
+            .with_rotation(Quat::from_rotation_y(-0.15 * PI)),
+        ..default()
+    });
+
+    // Spawn the light.
+    commands.spawn(DirectionalLightBundle {
+        directional_light: DirectionalLight {
+            illuminance: 15000.0,
+            shadows_enabled: true,
+            ..default()
+        },
+        transform: Transform::from_rotation(Quat::from_euler(
+            EulerRot::ZYX,
+            0.0,
+            PI * -0.15,
+            PI * -0.15,
+        )),
+        cascade_shadow_config: CascadeShadowConfigBuilder {
+            maximum_distance: 3.0,
+            first_cascade_far_bound: 0.9,
+            ..default()
+        }
+        .into(),
+        ..default()
+    });
+}
+
+/// Spawns the camera, including the [`ChromaticAberration`] component.
+fn spawn_camera(commands: &mut Commands, asset_server: &AssetServer) {
```

Comment:
Tiny nitpick, but I think this should be move a bit higher up because it contains the important bits for post processing to works. Spawning the scene is less relevant for someone that just wants to set it up for their scene.